docs/dev/reference/apis/sessions.md

```diff
@@ -60,6 +60,118 @@ If a resource was used by client A and then by client B, and client A disconnect
 
 A disconnected client will attempt to establish a new session immediately prior to the next operation it performs.
 
+#### Change the session timeout
+
+The default session length is 20 seconds.
+To change the session timeout length, pass the `timeout` parameter to the `DialOptions` object:
```

Comment:
```suggestion
To change this, pass the `timeout` parameter to the `DialOptions` object:
```
[opt] for brevity; no need to repeat esp. since `timeout` is a somewhat descriptive parameter name